README.md

```diff
@@ -34,12 +42,19 @@ npx jest MyObs
 npx jest -t accessibility
 ```
 
-We currently have two kinds of tests:
+### E2E tests
+We're using [Detox](https://wix.github.io/Detox/docs/19.x/) for E2E tests. If you want to run the e2e tests on your local machine (MacOS only), make sure you fulfill the RN development requirements, see above, and also follow the test-specific [environment setup](https://wix.github.io/Detox/docs/19.x/introduction/ios-dev-env/).
+
+```bash
+# Build the app and run the tests
+npm run e2e
+```
+
+If you have built the app already for a previous test, and just want to run an updated test without changing the app code, you can run `npm run e2e:test`.
 
-1. `tests/integration`: Tests the test the integration of multiple modules, e.g. a list of observation that makes requests to a mocked API, persists the response data in local storage, retrieves the data from local storage and renders components.
-1. `tests/unit`: Tests that only test specific modules, like a single component, or a hook.
+If you are running into some issues after the tests have been working for some time, try updating `applesimutils` with `brew update && brew upgrade applesimutils`.
 
-We're using [Jest](https://jestjs.io/) and [React Native Testing Library](https://callstack.github.io/react-native-testing-library/) for most of our tests, [factoria](https://github.com/phanan/factoria) and [faker.js](https://github.com/Marak/faker.js/) to generate test data for use with mocks. `Local*` model factories represent locally persisted data, while `Remote*` factories represent that kinds of records we get from an API or external package.
+If you want to run the Android tests make sure you have the correct [emulator setup](https://wix.github.io/Detox/docs/19.x/introduction/android-dev-env), and the emulator has the same name as in the .detoxrc.json file. You can then run `npm run e2e:build:android && npm run e2e:test:android`.
```

Comment:
Hm, I already had the system images installed, so I just created a new AVD in Android Studio. I tried to use the command line now to check the command line setup as described in the detox docs, but I get a Java error. So, should we just copy your commands into the Readme because they worked for you. (I don't remember if I installed the AOSP system image at one point or it came with the recent version of Android Studio).

For sure I would need to figure out the command line setup if we want to use it in GitHub Actions workflow as well.